client/app/components/project-geography.js

```diff
@@ -10,27 +10,27 @@ export default class ProjectGeographyComponent extends Component {
   // represents array of bbl models
   @tracked bbls;
 
-  // TODO: ISSUE #110
   // Used to find corresponding addresses for list of bbls.
   // Populated later with labs-search addon response.
-  // bblToAddressLookup = {};
+  @tracked bblToAddressLookup = {};
 
   @action
   selectSearchResult(labsSearchResult) {
     // `labsSearchResult` is object with `label` (address) and `bbl` attributes.
     // labsSearchResult.bbl comes back as a number,
     // whereas dcpBblnumber in CRM is stored as a string.
+    const currentBbl = labsSearchResult.bbl.toString();
+
     const bblObjectToPush = this.store.createRecord('bbl', {
-      dcpBblnumber: labsSearchResult.bbl.toString(),
+      dcpBblnumber: currentBbl,
       dcpDevelopmentsite: null,
       dcpPartiallot: null,
     });
 
-    // TODO: ISSUE #110
-    // set local variables for displaying address next to bbl in list
-    // const currentAddress = labsSearchResult.label;
-    // const formattedAddress = currentAddress.replace(', New York, NY, USA', '');
-    // this.bblToAddressLookup[currentBbl] = formattedAddress;
+    // set local variables for displaying address next to bbl in bbls list
+    const currentAddress = labsSearchResult.label;
+    const formattedAddress = currentAddress.replace(', New York, NY, USA', '');
+    this.bblToAddressLookup[currentBbl] = formattedAddress;
```

Comment:
I see! So you're storing all the address references in the a hash on the component. 

What if you stored them as a regular property on the model itself? For example, in pas-form model:

```
class PasForm extends Model {
  // other stuff...

  temporaryAddressLabel = '';
}
```

It won't have an `@attr` decorator, meaning it won't get serialized and sent to the server. It's just a property on the object.

Then, in your template, you can just reference it:

```handlebars
      {{#if bbl.temporaryAddressLabel}}
        ({{bbl.temporaryAddressLabel}})
      {{/if}}
```

Then, all you'd have to do is change this code to set it:
```javascript
  bblObjectToPush.temporaryAddressLabel = formattedAddress;
```

Thoughts? I think it's probably cleaner just because we aren't having to have the component maintain a stateful object lookup.

Reply:
ooh yes great idea, I'll update